Makefile

```diff
@@ -554,6 +554,24 @@ else
 	echo "Versions are different across Makefiles. Please ensure to keep them uniform."
 endif
 
+
+.PHONY: install-yamllint
+install-yamllint: ## Install yamllint if not present
+	@which yamllint > /dev/null || (echo "Installing yamllint..." && pip install --user yamllint)
```

Comment:
whats the guarantee that the pip will be available,  aren't there any alternative to download this other than pip install, may be via go?